### ISSUE TITLE
Figure out how to list samples in the data portal

### ISSUE THREAD

**Issue by pvangay** [microbiomedata/nmdc-metadata#349]
Related to https://github.com/microbiomedata/nmdc-metadata/issues/342

Samples are currently listed by study/proposal names, which can be misleading in some cases.

In 5/21/21 "docuthon" meeting, @mslarae13 @dehays @dwinston discussed options, which include one or more of:

- sample_name (new datasets)
- IGSN
- UUID
- local sample ID

Action items: 

- [ ] identify who needs to be included in the discussion / decision making
- [ ] schedule meeting to determine an alternative to what we're doing currently
- [ ] implement new alternative

Adding as listeners: @kfagnan @turbomam

**Comment by dehays** [comment]
This issue might be better off in nmdc-server (where, non intuitively, the client code is) because this is about how best to display an NMDC biosample.  That display can only use the biosample attributes that are available.  So this is not a question of whether to display an IGSN or a UUID, biosample has an id attribute that can take several forms: gold:, igsn: or emsl:.  For gold: and igsn: sample ids the ID could be displayed to link back to the record indicated by the identifier.  sample_name is a field on the MIxS 6, which would likely map to the NMDC biosample name field.  The NMDC biosample name field is what is currently displayed in the UI.

**Comment by ssarrafan** [comment]
@dehays should this issue be added to the July sprint board?

**Comment by dehays** [comment]
@ssarrafan Yes - and there could be two tasks here.  1) Given the attributes of a biosample in NMDC, how should that biosample be displayed in the UI; i.e. what field values should be displayed and how should they be displayed.  2) There is also discussion about whether the GOLD biosample name should be used as the NMDC biosample name.  GOLD biosample names default to a naming scheme that uses the sequencing project name combined with the researcher sample name.  If we pursue this modification for name from GOLD to NMDC then there needs to be a naming scheme defined.
